### DISCUSSION TITLE
How changed base path

### DISCUSSION THREAD

**Discussion by JOROZCOF206** [primefaces/community#743] [PrimeNG Templates] [answered]
Hello guys, I bought Apollo Template, so far so good. In this moment I have cleaning my template, deleting modules, deleting unnecessary things, so. I have a question, I changed the base path, that in this case is DASHBOARD, I do chaged for my base component, but when I make the change, it does not recognize the other routes, it redirects me 404. I have been checking all the modules, but I can't find how to change this base route. 

This is base path for default.

{
              path: '',
                loadChildren: () => import('./demo/components/dashboards/dashboards.module').then(m => m.DashboardsModule)
            },
But I changed for this: 
{
                path: '',
                loadChildren: () => import('./components/home/home.module').then(m => m.HomeModule)
            },
Other paths, redirect to 404. Only work it, if I add some letter o word to new path

Some help?

**Comment by cetincakiroglu** [comment, marked as answer]
Hi,

Sorry for the late response. Code below works and updates the base path

```
// app-routing.module.ts

const routes: Routes = [
   {
      path: '',
      component: AppLayoutComponent,
      children: [
        {
           path: '',
           loadComponent: () => import('./home/home.component').then((m) => m.HomeComponent),
        }
      ]
   }
]
```


```
// home.component.ts

import { CommonModule } from '@angular/common';
import { Component } from '@angular/core';

@Component({
    selector: 'app-home',
    standalone: true,
    imports: [CommonModule],
    template: `<h1>hello world</h1>`,
})
export class HomeComponent {}

```

Code above updates the base path and banking dashboard will redirect you to `404` page. Because, it seeks for the dashboards module to find banking. You also need to update `menu.json` and re organize your paths as well.
